### PR TITLE
Added wait funcion for async service operations on brain tests and disable mongodb test temporarily

### PR DESCRIPTION
**Pull request by f0rmiga**
## Description

Service creation and deletion are async and requires proper waiting.
Also, disabling the mongodb brain test temporarily.

## Test plan

Brain tests should pass.

### CODE REVIEW

**Comment by f0rmiga** [comment]
[brains: 016_minibroker_mongodb_test.rb failing often](https://trello.com/c/eW34pCo4/960-brains-016minibrokermongodbtestrb-failing-often)

**Comment by mook-as** [comment]
FWIW, there's similar code in https://github.com/SUSE/scf/pull/2143/commits/87741630b473995302952f08e29b6b9c547552bc — we probably want to only take one of the two (the other hasn't merge yet either).

File: src/scf-release/src/acceptance-tests-brain/test-scripts/016_minibroker_mongodb_test.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env ruby
 
+Process.exit(0)
+
```
Comment (mook-as) [inline src/scf-release/src/acceptance-tests-brain/test-scripts/016_minibroker_mongodb_test.rb]: I'd rather we rename the file to not match the default `INCLUDE` filter; that way, we can look at it by mentioning it explicitly.

Alternatively, modify the default `EXCLUDE` filter to include a keyword (and rename to match), maybe…

File: src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb [hunk 2]
```diff
@@ -150,3 +150,27 @@ def wait_for_namespace(namespace, sleep_duration=10)
         sleep sleep_duration
     end
 end
+
+def wait_for_async_service_operation(service_instance_name, retries=0)
+    service_instance_guid = capture("cf service --guid #{service_instance_name}")
+    return { :success => false, :reason => :not_found } if service_instance_guid == 'FAILED'
```
Comment (mook-as) [inline src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb]: FWIW, this is the same as `{success: false, reason: :not_found}`

Comment (f0rmiga) [reply src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb]: Thanks for pointing that out.

File: src/scf-release/src/acceptance-tests-brain/test-scripts/016_minibroker_mongodb_test.rb (hunk 1 above)
Comment (f0rmiga) [reply src/scf-release/src/acceptance-tests-brain/test-scripts/016_minibroker_mongodb_test.rb]: I got a fix for this test. I'd rather merge this PR to unblock the rest of open PRs and I'll open another PR with the mongodb fix in a bit.

File: src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb [hunk 3]
```diff
@@ -128,7 +133,14 @@ def run_test
 
             File.open("#{tmpdir}/service-params.json", 'w') { |f| f.puts service_params.to_json }
             run "jq -C . #{tmpdir}/service-params.json"
+            started_service_creation = Process.clock_gettime(Process::CLOCK_MONOTONIC)
```
Comment (mook-as) [inline src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb]: Not sure we need that hard core sort of timing, but sure

File: src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb [hunk 4]
```diff
@@ -150,3 +150,27 @@ def wait_for_namespace(namespace, sleep_duration=10)
...
+    attempts = 0
+    loop do
+        puts "# Waiting for service instance #{service_instance} to complete operation..."
+        service_instance_info = cf_curl("/v2/service_instances/#{service_instance_guid}")
+        return { :success => true } unless service_instance_info.key?('entity')
```
Comment (mook-as) [inline src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb]: In ruby, people tend to just say `… unless service_instance_info['entity']` as a test (and rely on `nil` being falsy)

File: src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb (hunk 3 above)
Comment (f0rmiga) [reply src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb]: 🤷

**Comment by f0rmiga** [comment]
Brain passes on Jenkins and CATS failures are not related to this PR. Merging.

**Pull request merged**